vk-video/src/vulkan_encoder/yuv_converter.rs

```diff
@@ -224,13 +217,30 @@ impl Converter {
         command_encoder.transition_resources(
             [].into_iter(),
             [wgpu::TextureTransition {
-                texture: &texture,
+                texture,
                 state: wgpu::TextureUses::RESOURCE,
                 selector: None,
             }]
             .into_iter(),
         );
 
+        self.device.wgpu_queue().submit([command_encoder.finish()]);
```

Comment:
IMO it shouldn't be like that. `transition_resources` and YUV conversion should be able to use the same command buffer without `submit` in between but it does not work otherwise